src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -30,18 +32,114 @@ public sealed class MonitoringConfiguration : ConfigurationCategory
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Metrics.RuntimeInstrumentation.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool RuntimeInstrumentationMetricsEnabled { get; set; }
+            
+            [Description("Indicates if metrics should be exported with the OpenTelemetry protocol.")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool OpenTelemetryProtocolExporter { get; set; }
+            
+            [Description("Endpoint where OpenTelemetryProtocol should sends data. Default: null (internal OTLP default settings).")]
+            [DefaultValue(null)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.Endpoint", ConfigurationEntryScope.ServerWideOnly)]
+            public string OtlpEndpoint { get; set; }
+            
+            [Description("Defines the protocol that OpenTelemetryProtocol should use to send data. Default: null (internal OTLP default settings).")]
+            [DefaultValue(null)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.Protocol", ConfigurationEntryScope.ServerWideOnly)]
+            public OtlpExportProtocol? OtlpProtocol { get; set; }
+            
+            [Description("OpenTelemetryProtocol custom headers. Default: null.")]
+            [DefaultValue(null)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.Headers", ConfigurationEntryScope.ServerWideOnly)]
+            public string OtlpHeaders { get; set; }           
+            
+            [Description("OpenTelemetryProtocol export processor type. Default: null.")]
+            [DefaultValue(null)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.ExportProcessorType", ConfigurationEntryScope.ServerWideOnly)]
+            public ExportProcessorType? OtlpExportProcessorType { get; set; }
+            
+            [Description("OpenTelemetryProtocol timeout value. Default: null.")]
+            [DefaultValue(null)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.Timeout", ConfigurationEntryScope.ServerWideOnly)]
+            public int? OtlpTimeout { get; set; }
+            
+            [Description("Indicates if metrics should be exported to the console output.")]
+            [DefaultValue(false)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ConsoleExporter", ConfigurationEntryScope.ServerWideOnly)]
+            public bool ConsoleExporter { get; set; }
+            
+            [Description("Expose instruments related to server storage.")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ServerWide.Storage.Enabled", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
In above config options names you use  ".Metrics.AspNetCoreInstrumentation." e.g. Monitoring.OpenTelemetry.Metrics.RuntimeInstrumentation.Enabled. 

Wouldn't it make sense to apply it here (and below as well)? Can we make the naming consistent?

Reply:
Also instead of `ServerWide` can we use just `Server`?

Reply:
I like the idea. 
I would go with:
e.g.
`Monitoring.OpenTelemetry.Meters.Server.Storage.Enabled`

Reply:
Also regarding this comment: https://github.com/ravendb/ravendb/pull/18589#discussion_r1658283298
For built-in:
`Indicates if AspNetCore metric is enabled or not. Default: true`
`Indicates if Runtime metric is enabled or not. Default: true`
And for ours:
I would suggest something like that:
`"Expose metric related to [NAME]."`